deploy/olm-catalog/cloud-resources/0.8.2/cloud-resources.v0.8.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,380 @@
+                - name: TAG_KEY_PREFIX
+                  value: integreatly.org/
+                image: quay.io/integreatly/cloud-resource-operator:v0.8.2
+                imagePullPolicy: Always
+                name: cloud-resource-operator
+                resources: {}
+              serviceAccountName: cloud-resource-operator
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - pods/exec
+          - services
+          - services/finalizers
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          verbs:
+          - '*'
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - '*'
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - apps
+          resourceNames:
+          - cloud-resource-operator
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          verbs:
+          - get
+        - apiGroups:
+          - apps
+          resources:
+          - replicasets
+          verbs:
+          - get
+        - apiGroups:
+          - integreatly
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        - apiGroups:
+          - integreatly.org
+          resources:
+          - '*'
+          - smtpcredentialset
+          - redis
+          - postgres
+          - redissnapshots
+          - postgressnapshots
+          verbs:
+          - '*'
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - prometheusrules
+          verbs:
+          - '*'
+        - apiGroups:
+          - config.openshift.io
+          resources:
+          - '*'
+          - infrastructures
+          - schedulers
+          - featuregates
+          - networks
+          - ingresses
+          - clusteroperators
+          - authentications
+          - builds
+          verbs:
+          - '*'
+        - apiGroups:
+          - cloudcredential.openshift.io
+          resources:
+          - credentialsrequests
+          verbs:
+          - '*'
+        serviceAccountName: cloud-resource-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - integreatly
+  maintainers:
+  - email: integreatly-support@redhat.com
+    name: Integreatly
+  maturity: alpha
+  provider:
+    name: Integreatly
+  replaces: cloud-resources.v0.7.1
```

Comment:
is this replace version correct?

Reply:
So this is the latest version in the integreatly operator thats on master, however on CRO master, its `v0.8.1`? Should I leave it at `v0.8.1` here in the PR, but manually change it in the integreatly operator?

Reply:
This requires a wider conversation on how we map CRO releases to the Integreatly operator. I would lean toward we stick with what ever is the current release of CRO, and we update this manually in the integreatly operator.

Reply:
Yeah.. I'd agree there, let me change it to whats in CRO master. `v0.8.1`